src/components/select/index.njk

```diff
@@ -150,7 +136,7 @@ The HTML <code>&lt;select&gt;</code> element represents a control that provides
     ],
     [
       {
-        text: 'label'
+        text: 'item.label'
```

Comment:
Hmmm should this be just `label`? It doesn't look like `items` uses `label` in the template.